copter/source/docs/tuning-process-instructions.rst

```diff
@@ -146,7 +274,7 @@ When oscillations start do not make large or sudden stick inputs. Reduce the thr
 5. Reduce the P term in steps of 10% until the oscillation disappears.
 6. Reduce the P term by a further 25%.
 
-Each time the P term is changed set the I term equal to the P term.
+Each time the P term is changed set the I term equal to the P term.  # TODO I term or I term change ?
 
```

Comment:
I think this line is fine as is. I am not sure what your question is there.

Reply:
We set I = P  or P percentage change = I percentage change ?

Reply:
After making a change to the P term set the I = P. If you are using Ch6 tuning you may not be able to set I = P until you land again. In any case it isn't important when I is set equal to P.